.github/workflows/release-drafter.yml

```
@@ -0,0 +1,18 @@
+name: release-drafter
+
+on:
+  push:
+    # branches to consider in the event; optional, defaults to all
+    branches:
+      - master
+      - "releases/**"
+      - "stable/**"
+
+jobs:
+  update_release_draft:
+    runs-on: ubuntu-20.04
+    steps:
+      # Drafts your next Release notes as Pull Requests are merged into "master"
```

Good work, just a very trivial nit:

```suggestion
      # Drafts your next release notes as Pull Requests are merged into "master"
```